sdks/python/apache_beam/testing/load_tests/combine_test.py

```diff
@@ -0,0 +1,101 @@
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+To run test on DirectRunner
+
+python setup.py nosetests \
+    --test-pipeline-options="
+    --input_options='{
+    \"num_records\": 300,
+    \"key_size\": 5,
+    \"value_size\":15,
+    \"bundle_size_distribution_type\": \"const\",
+    \"bundle_size_distribution_param\": 1,
+    \"force_initial_num_bundles\": 0
+    }'" \
+    --tests apache_beam.testing.load_tests.combine_test
+
+To run test on other runner (ex. Dataflow):
+
+python setup.py nosetests \
+    --test-pipeline-options="
+        --runner=TestDataflowRunner
+        --project=...
+        --staging_location=gs://...
+        --temp_location=gs://...
+        --sdk_location=./dist/apache-beam-x.x.x.dev0.tar.gz
+        --input_options='{
+        \"num_records\": 1000,
+        \"key_size\": 5,
+        \"value_size\":15,
+        \"bundle_size_distribution_type\": \"const\",
+        \"bundle_size_distribution_param\": 1,
+        \"force_initial_num_bundles\": 0
+        }'" \
+    --tests apache_beam.testing.load_tests.combine_test
+
+"""
+
+import unittest
+import json
+import logging
+import apache_beam as beam
+from apache_beam.testing import synthetic_pipeline
+from apache_beam.testing.test_pipeline import TestPipeline
+
+
+class CombineTest(unittest.TestCase):
+  def parseTestPipelineOptions(self):
+    return {
+        'numRecords': self.inputOptions.get('num_records'),
+        'keySizeBytes': self.inputOptions.get('key_size'),
+        'valueSizeBytes': self.inputOptions.get('value_size'),
+        'bundleSizeDistribution': {
+            'type': self.inputOptions.get(
+                'bundle_size_distribution_type', 'const'
+            ),
+            'param': self.inputOptions.get('bundle_size_distribution_param', 0)
+        },
+        'forceNumInitialBundles': self.inputOptions.get(
+            'force_initial_num_bundles', 0
+        )
+    }
+
+  def setUp(self):
+    self.pipeline = TestPipeline(is_integration_test=True)
+    self.inputOptions = json.loads(self.pipeline.get_option('input_options'))
+
+  class _get_element(beam.DoFn):
+    def process(self, element):
+      yield element
+
+  def testCombineGlobally(self):
+    with self.pipeline as p:
+      # pylint: disable=expression-not-assigned
+      (p
+       | beam.io.Read(synthetic_pipeline.SyntheticSource(
+           self.parseTestPipelineOptions()))
+       | 'Combine with Top' >> beam.CombineGlobally(
+           beam.combiners.TopCombineFn(1000))
+       | 'Consume' >> beam.ParDo(self._get_element())
+      )
```

Comment:
@pabloem here I changed `combined` method to `Top` (well to `TopCombineFn `, but tell me if you think it's better to have just `Top` with a chain of operations).
Also ParDo is actually NoOpsPardo, because I wasn't sure about what can be good consumer - maybe `len`?

Reply:
Maybe get_element is enough